frontend/src/lib/components/PropertyKeyInfo.tsx

```diff
@@ -376,15 +380,31 @@ export const keyMapping: KeyMappingInterface = {
             examples: ['Australia/Sydney', 'Asia/Kolkata'],
         },
         $geoip_subdivision_1_name: {
-            label: 'Subdivision Name',
+            label: 'Subdivision 1 Name',
             description: `Name of the subdivision matched to this event's IP address.`,
             examples: ['New South Wales', 'Tamil Nadu'],
         },
         $geoip_subdivision_1_code: {
-            label: 'Subdivision Code',
+            label: 'Subdivision 1 Code',
             description: `Code of the subdivision matched to this event's IP address.`,
             examples: ['NSW', 'TN'],
         },
+        $geoip_subdivision_2_name: {
+            label: 'Subdivision 2 Name',
```

Comment:
I'm not sure about this, I know this is the official definition but it's just so unfriendly. I would just do something "State / Province", "City" & "Municipality / Town", and maybe add a link to the docs where we explain in more detail and maybe link to something like [this](https://en.wikipedia.org/wiki/List_of_administrative_divisions_by_country).

L1, L2, L3 is subdivision terminology I believe is quite domain-specific, will be quite confusing.

Reply:
Well, this is not hierarchical (1, 2, 3 are just indexes), and almost all IP addresses have at least one subdivision in Maxmind, but few have two and very few have three. I believe there are practically none with four. For instance I added a Brooklyn example to address the other comment, and it only has one subdivision: "New York". So I don't think this is something to be improved really (now what would be better would be a property of type `string[]`, but we are currently outright unable to filter on such properties, so avoiding that)

Reply:
Oh I see these are not the administrative regions. Yeah I think this is fine for now, though tbh I would considering dropping support for viewing these properties altogether. I don't think the cost of the extra mental burden of this is actually worth the value we can provide. As you mentioned there's also little IPs with this information, and I also think this might be a lot less accurate.